schemas/catalog/20210202.json

```diff
@@ -0,0 +1,398 @@
+                        },
+                        "nft": null
+                    }
+                }
+            ],
+            "required": [
+                "version",
+                "title",
+                "artist",
+                "notes",
+                "duration",
+                "mimeType",
+                "trackNumber",
+                "project",
+                "artwork",
+                "visualizer"
+            ],
+            "properties": {
+                "version": {
+                    "type": "string",
+                    "description": "Calendar version of the schema so that consumers can correctly parse the json",
+                    "const": "catalog-20210202"
+                },
+                "title": {
+                    "type": "string",
+                    "description": "Title of the track",
+                    "examples": ["Our Taproot"]
+                },
+                "artist": {
+                    "type": "string",
+                    "description": "Name of the artist who created the track",
+                    "minLength": 1,
+                    "examples": ["Omari Jazz"]
+                },
+                "notes": {
+                    "type": ["string", "null"],
+                    "description": "An optional property for describing the track",
+                    "examples": ["Dreams are avatars of the subconscious", null]
+                },
+                "duration": {
+                    "type": "number",
+                    "description": "Length of the audio file in seconds (must be > 1ms)",
+                    "minimum": 0.001,
+                    "examples": [65.881]
+                },
+                "mimeType": {
+                    "type": "string",
+                    "description": "MimeType of the audio file. Only lossless formats (aif, wav, flac) are supported.",
+                    "enum": ["audio/x-aiff", "audio/aiff", "audio/wav", "audio/x-wav", "audio/flac"],
+                    "examples": ["audio/aiff"]
+                },
+                "trackNumber": {
+                    "type": ["integer", "null"],
+                    "description": "The place which the track appears in its project (e.g. track 4 off an album)",
+                    "minimum": 1,
+                    "examples": [1, 9, null]
+                },
+                "project": {
+                    "type": ["object", "null"],
+                    "description": "Describes the body of work the record is a part of (e.g. an album, EP, or compilation)",
+                    "required": [
+                        "title", 
+                        "artwork"
+                    ],
+                    "examples": [
+                        {
+                            "title": "Dream Child",
+                            "artwork": {
+                                "isNft": false,
+                                "detail": {
```

Comment:
`nit`: this example is inconsistent with the artwork type definition lower in the schema.